### ISSUE TITLE
how use ffmpeg by Gpu geforce GTX 

### ISSUE THREAD

**Issue by saman-magnus** [MacWarrior/clipbucket-v5#202]
hello i try Million way's to find a way for encoding video file ffmpeg but i can't find any , i search at google but nothing, another problem ffmpeg use to much cpu power , what can i do any idea ???
+ i use geforce GTX 1070 ti and geforce GTX 960 turbo 
![cpuhard](https://user-images.githubusercontent.com/41460065/150348422-54093ac0-ea42-4b91-8a25-4f1d1ad7bf20.png)

**Comment by MacWarrior** [comment]
Hello @saman-magnus,

I've also spend time looking for GPU conversion... and it require to recompile FFMpeg with a specific argument (nv-codec-hearders)

Check this : 
https://docs.nvidia.com/video-technologies/video-codec-sdk/ffmpeg-with-nvidia-gpu/

After finalising HLS conversion, I'll have a huge conversion process to do and I'll probably look further into this option to massively improve conversion time (Here with RTX3080 / GTX1080)
If things go well, I'll make a tuto / script to automate ffmpeg compilation

Best regards,
MacWarrior.

**Comment by saman-magnus** [comment]
this GeForce Gtx 960 Turbo Gpu and i waiting for ur job bro 
![cmd](https://user-images.githubusercontent.com/41460065/150372379-75c2b32a-80c2-424c-8dd2-c0d3e2e422cd.png)

**Comment by saman-magnus** [comment]
hello MacWarrior  i find a way to fix cpu usage and use GPU for Encode ffmpeg, i was install_
https://docs.nvidia.com/video-technologies/video-codec-sdk/ffmpeg-with-nvidia-gpu/ on GeForce GTX 960 T
after that i change video encode from H264 to H264_nvenc and finally i see this one :
![cpu lov](https://user-images.githubusercontent.com/41460065/150667290-ec9f8684-203a-4e0a-b82f-6c844df8ed14.png)
so its much faster and low cpu usage :+1:  what do you think ?

**Comment by anhthoai** [comment]
> hello MacWarrior i find a way to fix cpu usage and use GPU for Encode ffmpeg, i was install_ https://docs.nvidia.com/video-technologies/video-codec-sdk/ffmpeg-with-nvidia-gpu/ on GeForce GTX 960 T after that i change video encode from H264 to H264_nvenc and finally i see this one : ![cpu lov](https://user-images.githubusercontent.com/41460065/150667290-ec9f8684-203a-4e0a-b82f-6c844df8ed14.png) so its much faster and low cpu usage 👍 what do you think ?

Yes. You just change libx264 to h264_nvenc for GPU in command at ffmpeg.class.php
